File: g211210055_labsystem_rn/src/config/databaseConfig.ts
// Veritabanı bağlantı ayarları
export class DatabaseConfig {
  // PostgreSQL bağlantı bilgileri
  // Bu bilgileri kendi veritabanınıza göre güncelleyin
  static readonly host = 'localhost';
  static readonly port = 5432;
  static readonly database = 'elab_system';
  static readonly username = 'postgres';
  static readonly password = 'Kenan21.'; // Kendi şifrenizi girin

  // API Base URL (Backend API servisi için)
  // Not: React Native'de direkt PostgreSQL bağlantısı yapamayız, bu yüzden bir backend API servisi gerekir
  // Bu URL'yi kendi backend API servisinize göre güncelleyin
  static readonly apiBaseUrl = 'http://localhost:3000/api'; // Örnek backend API URL
}

File: config/database_config.dart
// Veritabanı bağlantı ayarları
class DatabaseConfig {
  // PostgreSQL bağlantı bilgileri
  // Bu bilgileri kendi veritabanınıza göre güncelleyin
  static const String host = 'localhost';
  static const int port = 5432;
  static const String database = 'elab_system';
  static const String username = 'postgres';
  static const String password = 'Kenan21.'; // Kendi şifrenizi girin

  // Bağlantı string'i
  static String get connectionString =>
      'postgresql://$username:$password@$host:$port/$database';
}
